app/controllers/users_controller.rb

```diff
@@ -2,13 +2,45 @@
 
 class UsersController < ApplicationController
   attr_reader :user
```

Comment:
We can ✂️ this.

So you need following here:
You need to set `@user` ivar in actions `edit,update` like this
```ruby
class UsersController < ApplicationController
   before_action :set_user, only: [:edit, :update]
   .....
   
  pirvate
     def set_user
        @user = User.find(params[:id])
     end
end
```

Now when you have `@user` rails will make it accessible to your view.

Also we can ✂️ all actions that are not used for now such as `new,...`